Packaging.props

```diff
@@ -1,9 +1,9 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <LicenseUrl>https://github.com/dotnet/standard/blob/master/LICENSE.TXT</LicenseUrl>
+    <LicenseUrl Condition="'$(PackageLicenseExpression)' == ''">https://github.com/dotnet/standard/blob/master/LICENSE.TXT</LicenseUrl>
```

Comment:
It seems odd to allow an outside location to override the license, no?

Reply:
This is just a temporary workaround needed until the Arcade SDK change is merged.